tests/app/forms/field_handlers/test_text_area_handler.py

```diff
@@ -73,3 +73,24 @@ def test_get_length_validator_with_max_length_override():
     validator = text_area_handler.get_length_validator()
 
     assert validator.max == 30
+
+
+def test_get_textarea_row_length():
+    answer = {
+        "guidance": "",
+        "id": "answer",
+        "rows": 3,
+        "label": "Enter your comments",
+        "mandatory": False,
+        "q_code": "0",
+        "type": "TextArea",
```

Comment:
We shouldn't use answer properties that aren't required here - `guidance` and `q_code` should be removed.